common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/AuthenticatorGetInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.line.auth.fido.fido2.common.mdsv3;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class AuthenticatorGetInfo {
+    private List<String> versions;
+    private List<String> extensions;
+    private String aaguid;
```

Comment:
OK.
Then, since the current `String aaguid` seems to be parsing the metadata provided by the MDS Server well, why don't we change it if we need it later?

Reply:
Ok understood :-)
That's why I'm a bit disappointed we have to rely on the CTAP spec for that structure ...a